### ISSUE TITLE
Adding docs and how-to

### ISSUE THREAD

**Issue by De-Panther** [De-Panther/unity-webxr-export#94]
There are no docs and tutorials about the different components and how to start a project or import the packages.

I think that creating a general docs folder at the root of the repo, and link from docs folders of the packages would be easier to maintain.

**Comment by HawkenKing** [comment]
I'm having trouble accessing the WebXR namespace from scripts inside /Assets/

Would be good to get a guide on this.

`Assets/myproject/PlayerController.cs(4,7): error CS0246: The type or namespace name 'WebXR' could not be found (are you missing a using directive or an assembly reference?)`

with

```using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using WebXR.Interactions;

public class PlayerController : MonoBehaviour
{
    public WebXRController leftController;
    public WebXRController rightController;

}```

**Comment by De-Panther** [comment]
If you are using VSCode, it might be related to the VSCode package version, updating it to 1.2.3 (or up) should solve your issue.
It's a Unity bug, so I don't think that we should have a doc for it. You will get the same error for trying to use `UnityEngine.UI`.

**Comment by De-Panther** [comment]
Your issue might also be related to the use of `.asmdef` file. And while it's Unity related issue, this is something that should be in the docs. Thanks!
For now, developers who also has similar issue, I made a video tutorial about this - https://youtu.be/N9iACzNCnos

**Comment by HawkenKing** [comment]
ah yes can confirm this, all working now

**Comment by TheXRMonk** [comment]
Adding to this, it would be really nice to have some more in-code comments in general. Especially on the functions and events in WebXRManager. 
For instance, I can't really wrap my head around the ViewerHitTest state. Guess it's related to [this](https://immersive-web.github.io/hit-test/)? And the XRChange event? what are those parameters? :octocat: 

Ah, just realized you are moving the cameras rather than the objects during hittest-updates. Guess that's because of static bathing optimization 👍

**Comment by De-Panther** [comment]
Yes, should add more comments. Would be happy if others would be able to help with that.

Regarding the `ViewerHitTest`, you can check the WebXR Interactions package for `SceneHitTest` which is one way to use it.
